core/src/impl/Kokkos_Utilities.hpp

```diff
@@ -415,7 +415,37 @@ struct inclusive_scan_integer_sequence {
 };
 
 template <typename T>
-using identity_t = T;
+struct identity {
+  using type = T;
+};
+
+template <typename T>
+using identity_t = typename identity<T>::type;
+
+//==============================================================================
+// <editor-fold desc="remove_cvref_t"> {{{1
+
+template <class T>
+struct remove_cvref : identity<typename std::remove_cv<
+                          typename std::remove_reference<T>::type>::type> {};
```

Comment:
Why would you use `identity` here?
```suggestion
struct remove_cvref : std::remove_cv_t<std::remove_reference_t<T>>::type> {};
```
Also please comment this is added in C++20

Reply:
does clang 11 have this in C++20 mode already?

Reply:
If it supports C++20 it has it.

Reply:
The suggestion from @dalg24 is cleaner, as we are deriving from another type trait, which already gives us the embedded `type` member.  `identity` makes more sense when we don't have that, because it is more declarative in that it allows the body of the type trait struct to be empty.

It is also cleaner because of less use of `typename`.  Many of the type traits from vendors aren't using the `_t` versions because those didn't appear until C++14 and they have to be backwards compatible to C++11.

Separate issue:  given that C++20 settled on the name `type_identity` for this, we should change ours too.